### ISSUE TITLE
How to make recaptcha on contact form mandatory?

### ISSUE THREAD

**Issue by MojoJojo** [ThemeFuse/Unyson#3681]
I'm using Unyson designer and extensions and have added recaptcha to my contact form. However,  I could not find option to make the recaptcha field mandatory and I am able to submit the form without verifying. Please see the screen shots below. The form can be accessed at the bottom of the page https://www.dukstra.com
To re-produce the issue, try filling up the form and not ticking the recaptcha validation field.

Not sure if its a bug or if I'm missing something?

**Comment by freefefes** [comment]
Any news?
I have the same problem!

**Comment by MojoJojo** [comment]
Nope. Feel somewhat cheated there. They asked me to use contact form 7 instead.

**Comment by danyj** [comment]
Not sure how guys but here , https://themezly.com/contact-us/
ours is same and you must check captcha, unless your theme dev did and override and 
messed up the form

**Comment by danyj** [comment]
And here I dont see captcha on 
https://www.dukstra.com/contacts/

**Comment by MojoJojo** [comment]
We removed it and replaced it with contact form based on suggestion by template-monster support (from whom I bought the theme) . Is there a way to trouble shooting/debug this?

**Comment by danyj** [comment]
without seeing the fw contact form with captcha on, no
